include/zephyr/net/wifi.h

```diff
@@ -237,6 +237,8 @@ const char *wifi_band_txt(enum wifi_frequency_bands band);
 #define WIFI_ENT_IDENTITY_MAX_LEN 64
 /** Max enterprise password length */
 #define WIFI_ENT_PSWD_MAX_LEN 128
+/** Max enterprise identity users */
+#define WIFI_ENT_IDENTITY_MAX_USERS 8
```

Comment:
This doesn't belong in this file, this is only for generic/protocol related stuff.

Reply:
and this should be part of Kconfig.

Reply:
Updated as comment